Denote instructions are for classic portal

andynava opened the pull request:
It took me a while to figure out why the instructions to create an MFA Provider were there twice. This change makes it clear by denoting "Classic Portal" in the section header

guyhay commented:
updates ready to go

stevestein commented:
looks good, you can merge. I'm going to do a quick pull request right after this to fix a typo...

guyhay commented:
Thanks, and I made one further edit to clarify eDTUs

From: Ross McAllister [mailto:notifications@github.com]
Sent: Monday, May 11, 2015 09:29
To: Azure/azure-content
Cc: Guy Haycock
Subject: Re: [azure-content] Update sql-database-elastic-pool.md (#3521)


Merged #3521<https://github.com/Azure/azure-content/pull/3521>.

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-content/pull/3521#event-301935007>.

andynava committed:
Denote instructions are for classic portal

It took me a while to figure out why the instructions to create an MFA Provider were there twice. This change makes it clear by denoting "Classic Portal" in the section title

MicrosoftGuyJFlo reviewed (commented):
Thank you for the edit but this section is being deprecated as everything has been moved to the new Azure portal.
